src/utilities/helpers/focus-element/index.ts

```diff
@@ -1,12 +1,15 @@
 /* eslint-disable import/prefer-default-export */
-export const focusElement = (el, focusVisible = false) => {
+/* eslint-disable consistent-return */
```

Comment:
Q: Why is this necessary? I mean, is it really necessary to return this Promise? If so, then maybe it'd be reasonable to return some promise in every case? And then this comment `/* eslint-disable consistent-return */` wouldn't be necessary :) ESlint rules may be useful ;)

Reply:
A promise is necessary. I will add a promise to be consistent in all cases and will remove the ESlint comment as you wrote 🙂